application/cms/models.py

```diff
@@ -102,18 +102,83 @@ def process(value):
 
 
 class FrequencyOfRelease(db.Model):
+    __tablename__ = "frequency_of_release"
```

Comment:
@samuelhwilliams are these not inferred automatically?

Reply:
Yes, but explicit is better than implicit, particularly when we need to reference these in code elsewhere.

Reply:
@samuelhwilliams heh, @TheDoubleK and I have removed these in the past, coming to the opposite conclusion.

I don’t mind either way really. In Ruby on Rails the convention is to rely upon the inference unless you need to override it.

Reply:
Python is the opposite: prefer being explicit. See Zen of Python: https://www.python.org/dev/peps/pep-0020/#id3

Reply:
@samuelhwilliams heh, I bow to your superior Python zen mastery. Rails (and to an extent, Ruby) has the opposite philosophy of convention over configuration: https://guides.rubyonrails.org/active_record_basics.html#convention-over-configuration-in-active-record

Interestingly, https://docs.sqlalchemy.org/en/latest/orm/tutorial.html suggests that `__tablename__` is required. But maybe that's because it’s inheriting from `declarative_base()`?

Reply:
Interesting. I've never known it as a required thing either. But I've never used the declarative base (knowingly). It's always been from `db.Model`.